src/lib/drivers/smbus/SMBus.hpp

```diff
@@ -48,7 +49,7 @@ class SMBus : public device::I2C
 {
 public:
 	SMBus(int bus_num, uint16_t address);
-	~SMBus() override = default;
+	~SMBus();
```

Comment:
It doesn't really matter, but you might as well keep the override, it ensures the function in the base class is virtual.

```suggestion
	~SMBus() override;
```